components/ws-manager-api/typescript/src/promisified-client.ts

```diff
@@ -34,6 +35,7 @@ async function linearBackoffRetry<Res>(run: (attempt: number) => Promise<Res>, a
         try {
             return await run(i);
         } catch (err) {
+            log.error(`ws-manager call error`, err);
```

Comment:
IMHO should be part of the `warn` log in line 51. Otherwise I'm afraid we're cluttering the logs.

Reply:
done